src/lib/layouts/DocsArticle.svelte

```diff
@@ -16,11 +16,22 @@
     import { scrollToTop } from '$lib/actions/scrollToTop';
 
     import { Feedback } from '$lib/components';
+    import { onMount } from 'svelte';
 
     export let title: string;
     export let toc: Array<TocItem>;
     export let back: string | undefined = undefined;
     export let date: string | undefined = undefined;
+
+    let reducedArticleSize = false;
+
+    onMount(() => {
+        // reduces size for articles using numbered sections.
+        const slotWrapper = document.querySelector('.slot-wrapper');
+        reducedArticleSize = !!slotWrapper?.querySelector(
+            '.web-article-content-section.is-with-line .web-numeric-badge'
+        );
+    })
```

Comment:
This still feels hacky. And what if they resize the window?

@TorstenDittmann, any ideas on how to best handle this?

Reply:
@stnguyen90 Works on resize. The issue only occurs at certain breakpoints which the css handles.

Reply:
But does it also work when I manually resize the window without a reload @ItzNotABug?

Reply:
> But does it also work when I manually resize the window without a reload @ItzNotABug?

Just verified on a tutorial docs page, seems to work.